features/projects/components/project-card/project-card.tsx

```diff
@@ -26,7 +26,7 @@ export function ProjectCard({ project }: ProjectCardProps) {
   const { name, language, numIssues, numEvents24h, status } = project;
 
   return (
-    <div className={styles.container}>
+    <div className={styles.container} data-testID={project.id}>
```

Comment:
🌵  There's like a weird warning/error react or the Playwright debugger is throwing at me about using a capitalized `ID`.

Playwright docs list the supported attributes: https://playwright.dev/docs/other-locators#id-data-testid-data-test-id-data-test-selectors

And while this works it's worth noting the warning, hence why it's a 🌵